1.8hm.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+
+//�������� ���������, ��������� ���������� ������� ��������� � �������.
+
+int nullCounter(int *arr, int len)

```

Comment:
```suggestion
int zerosCounter(int *arr, int len)
```
Так более понятно. В C есть константа `NULL`, по названию кажется, что ты такие значения считаешь.